posthog/api/insight_serializers.py

```diff
@@ -189,3 +224,7 @@ class FunnelSerializer(GenericInsightsSerializer, BreakdownMixin):
     funnel_window_days = serializers.IntegerField(
         help_text="(DEPRECATED) Funnel window size in days.", required=False, default=14
     )
+    breakdowns = BreakdownField(
+        required=False,
+        help_text='**Experimental**. Alternative to `breakdown`, where you can pass through an object with different types of breakdowns.\nFor example: `[{property: 291, type: "cohort"}, {property: "email", type: "person"}]`',
```

Comment:
Yep, deffo experimental where I last touched it 

(I don't know if @hazzadous moved it on at all although he might have)

It was aimed at allowing mixed breakdowns but the underlying queries assume all breakdowns are the same time... 

so `[{property:"email", type: "person", value: "a", operator: "contains"}, {property: "email", type: "person", value: "b", operator: "contains"}]`

(assuming I've remembered the operator names correctly)

Reply:
I didn't change to support mixed breakdowns in the implementation.

Reply:
Ah, in that case should we roll back this feature flag? It's quite buggy and currently rolled out to beta users: https://app.posthog.com/feature_flags/1119.

I'll remove this field

Reply:
Definitely if it's only delivering pain